clang/test/CodeGenSYCL/device_has.cpp

```diff
@@ -6,38 +6,46 @@
 using namespace sycl;
 queue q;
 
-// CHECK: define dso_local spir_kernel void @{{.*}}kernel_name_1{{.*}} !sycl_declared_aspects ![[ASPECTS1:[0-9]+]] !srcloc ![[SRCLOC1:[0-9]+]]
+// CHECK-DAG: define dso_local spir_kernel void @{{.*}}kernel_name_1{{.*}} !sycl_declared_aspects ![[ASPECTS1:[0-9]+]] !srcloc ![[SRCLOC1:[0-9]+]]
```

Comment:
The definition of `func10` always before appeared as the first definition in the output (I'm unsure why), and I thought it would be a little awkward to put the check for `func10` at the top of the file separated from its definition, so I changed the CHECKs to CHECK-DAG.

Reply:
Oh I see. Weird. Thanks for clarifying. Are you sure it was `func10` and not `func1`. Anyway it doesn't really matter. I think the test is robust with CHECK-DAG also